openverse_catalog/dags/providers/provider_api_scripts/inaturalist.py

```diff
@@ -201,7 +215,7 @@ def compare_update_dates(
         raise AirflowSkipException("Nothing new to ingest")
 
     @staticmethod
-    def load_catalog_of_life_names(remove_api_files: bool):
+    def load_catalog_of_life_names(task: PythonOperator, remove_api_files: bool):
```

Comment:
I believe the typing on this would be:
```suggestion
    def load_catalog_of_life_names(task: TaskInstance, remove_api_files: bool):
```
Of `airflow.models.TaskInstance`

Reply:
This I'm pretty confident about. A task _instance_, does not necessary have the `execution_timeout`. That piece of information lives in the task (type = operator), or in the dag (type DAG, where it's the default for tasks within that dag), but not every task_instance object (at least when I was testing things) has all of the task info associated with it and not ever task has all of the dag info associated with it. I think that some of the challenge is that these are ultimately in different tables in a relational database and they don't always come together as a package when they're in python.  That's why I have `task` in the parameters rather than `ti`, and in some cases, both `task` and `ti`. :/

Reply:
AAHHHHH, okay my mind spaced and I thought this was `task_instance` - didn't realize you could request `task` from the airflow context too!